web_external/Featuresets/FeaturesetView.js

```diff
@@ -19,9 +23,45 @@ isic.views.FeaturesetView = isic.View.extend({
 
     render: function () {
         this.$el.html(isic.templates.featuresetPage({
-            featureset: this.model
+            featureset: this.model,
+            currentUser: girder.currentUser
         }));
 
         return this;
+    },
+
+    confirmDestroy: function () {
+        girder.confirm({
+            text: '<h4>Permanently delete <b>"' + _.escape(this.model.name()) + '"</b> featureset?</h4>',
+            escapedHtml: true,
+            confirmCallback: _.bind(function () {
+                // Ensure dialog is hidden before continuing. Otherwise,
+                // when destroy() displays its modal alert dialog,
+                // the Bootstrap-created element with class "modal-backdrop"
+                // is erroneously not removed.
+                $('#g-dialog-container').on('hidden.bs.modal', _.bind(this.destroy, this));
+            }, this)
+        });
+    },
+
+    destroy: function () {
+        this.model.destroy({
+            success: function (model, resp, options) {
+                girder.confirm({
+                    text: '<h4>Featureset <b>"' + _.escape(model.name()) + '"</b> deleted</h4>',
+                    escapedHtml: true,
+                    yesClass: 'hidden',
+                    noText: 'OK'
+                });
+            },
+            error: function (model, resp, options) {
+                girder.confirm({
+                    text: '<h4>Error deleting featureset</h4><br>' + _.escape(resp.responseJSON.message),
+                    escapedHtml: true,
+                    yesClass: 'hidden',
+                    noText: 'OK'
+                });
```

Comment:
I added `isic.showAlertDialog` for informational modal dialogs and it makes sense to use it in place of these `girder.confirm`s.